app/src/common/translations/en.json

```diff
@@ -59,10 +60,6 @@
     "cryptocurrencies": {
       "name": "Cryptocurrencies",
       "headline": "",
-      "configuration": {
```

Comment:
I think you've removed this by accident - can you please have another look?

Other than this, the PR looks good, thanks!